front/transformers/AccountTransformer.js

```diff
@@ -34,12 +34,22 @@ export default class AccountTransformer extends ApiTransformer {
     }
 
     let data = _.get(item, 'attributes')
```

Comment:
Few recomandations. Make sure JS variables are written in camelCase, not uses abbreviations unless the name is way too long and shoot for the shorter code variants :)

ex. 
```
let creditCardType = get(data, 'credit_card_type') ?? 'monthlyFull'
```

Reply:
Yeah makes sense, thank you! :)